openedx/features/course_experience/tests/views/test_course_outline.py

```diff
@@ -159,7 +246,33 @@ def test_resume_course(self):
         self.assertContains(response, 'Resume Course', count=2)
 
         content = pq(response.content)
-        self.assertTrue(content('.action-resume-course').attr('href').endswith('/sequential/' + section.url_name))
+        self.assertTrue(content('.action-resume-course').attr('href').endswith('/sequential/' + sequential2.url_name))
+
+    def test_resume_course_deleted_sequentials(self):
+        """
+        Tests resume course when the last accessed sequential is deleted and
+        there is are so sequentials left in the vertical.
```

Comment:
Fixed this and "mark_lasted_accessed" type.  I am just amending my commit because I want tests to pass as early as possible.